input/docs/cs/search_circles.md

```diff
@@ -0,0 +1,55 @@
+title: Hledání kružnic - Houghova transformace.
+---
+>Tento dokument pracuje s obrázkem: 
+
+* [Obrázek oka](../media/oko.jpg) in `A` 
+# Načtení obrázku jeho předzpracování a hledání hran
```

Comment:
lehké gramatické záležitosti - i v komentářích (nicméně ty jsou zase užitečné, takže fajn), ale to už necháme být..